priv/repo/migrations/20190819204535_add_avatar_listings_removed_state.exs

```diff
@@ -0,0 +1,8 @@
+defmodule Ret.Repo.Migrations.AddAvatarListingsRemovedState do
+  use Ecto.Migration
+  @disable_ddl_transaction true
+
+  def change do
+    Ecto.Migration.execute("ALTER TYPE avatar_listing_state ADD VALUE IF NOT EXISTS 'removed'")
```

Comment:
should we update the state of the existing records here to go from `delisted` to `removed`?

Reply:
Hmm yeah that probably makes sense